panther_utils/include/panther_utils/common_utilities.hpp

```diff
@@ -104,6 +105,22 @@ std::fstream OpenFile(const std::string & file_path, const std::ios_base::openmo
   return file;
 }
 
+/**
+ * @brief Checks if required version is satisfied.
+ *
+ * @param version Version to be verified.
+ * @param required_version The required version.
+ * @return true if the required version is satisfied, false otherwise.
+ */
+inline bool MeetsVersionRequirement(const float version, const float required_version)
+{
+  if (std::isnan(version) || std::isnan(required_version)) {
+    return false;
+  }
+
+  return version >= required_version - std::numeric_limits<float>::epsilon();
```

Comment:
Why not like this:

```suggestion
  return version >= required_version;
```

Reply:
It's the case of floating point arithmetics. IMO your suggestion is good too, but I didn't see a need to change it compared to the original version.